features/step_definitions/create_facility_steps.rb

```diff
@@ -5,3 +5,7 @@
 Then(/^I check Available check box$/) do
   check 'facility_status'
 end
+
+Given(/^I am on the facility show page$/) do
+  visit facilities_path
```

Comment:
I mean that `facility show page` implies a visit to `facilities_path(12)` or the like.